ios/01/_bugsnag_arm64.crash

```diff
@@ -1,30 +1,27 @@
-<span class="cp-wrong">App locks up.</span>
-
-Segmentation fault: 11: 
-
-bugsnag/BugsnagEvent.m:91:0 -[BugsnagEvent getStackTraceWithException:]
-bugsnag/BugsnagEvent.m:50:0 -[BugsnagEvent addSignal:]
-bugsnag/BugsnagNotifier.m:73:0 -[BugsnagNotifier notifySignal:]
-bugsnag/Bugsnag.m:67:0 handle_signal
-libsystem_platform.dylib:0 _sigtramp
-libsystem_pthread.dylib:0 pthread_getname_np
-libsystem_pthread.dylib:0 pthread_getname_np
-<span class="cp-wrong">CrashLibiOS:0 -[CRLCrashAsyncSafeThread crash] | Missing filename, and line number</span>
-CrashProbe iOS/CRLDetailViewController.m:53:0 -[CRLDetailViewController doCrash]
-UIKit:0 -[UIApplication sendAction:to:from:forEvent:]
-UIKit:0 -[UIControl _sendActionsForEvents:withEvent:]
-UIKit:0 -[UIControl touchesEnded:withEvent:]
-UIKit:0 _UIGestureRecognizerUpdate
-UIKit:0 -[UIWindow _sendGesturesForEvent:]
-UIKit:0 -[UIWindow sendEvent:]
-UIKit:0 -[UIApplication sendEvent:]
-UIKit:0 _UIApplicationHandleEventFromQueueEvent
-UIKit:0 _UIApplicationHandleEventQueue
-CoreFoundation:0 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__
-CoreFoundation:0 __CFRunLoopDoSources0
-CoreFoundation:0 __CFRunLoopRun
-CoreFoundation:0 CFRunLoopRunSpecific
-GraphicsServices:0 GSEventRunModal
-UIKit:0 UIApplicationMain
-CrashProbe iOS/main.m:16:0 main
-libdyld.dylib:0 start
+Exception Type:  EXC_BAD_ACCESS (SIGSEGV)
+Exception Subtype: KERN_INVALID_ADDRESS
+Attempted to dereference garbage pointer 0x1.
+                                                                                                  
+0  libsystem_platform.dylib _platform_memmove                                                 
+1                           pthread_getname_np                                                
```

Comment:
This looks like an invalid frame to me. It shows no binary image and the method name is identical to frame 2, which isn't possible. Even though the apps frames are right, it should be highlighted that this is not correct. Maybe even flag this as incorrect, as the stack trace is not 100% correct.